resume.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>wommy.io - resume</title>
+
+		<link rel='stylesheet' href='style.css'>
+		<style>
+			#main {
+				margin: 1rem;
+			}
+			h2 {
+				border-bottom: 1px solid silver;
+				padding-bottom: .3rem;
+			}
+			.margin{
+				margin: 1rem 0;
+			}
+			.indent{
+				padding-left: 1.25rem;
+			}
+			.highlight{
+				background-color: gainsboro;
+				border-radius: 3px;
+				padding: .2em .4em;
+				font-size: 85%;
+				font-family: monospace;
+			}
+		</style>
```

Comment:
So in this one, you have an external stylesheet and inline styles. I assume there are some global styles that are externally linked, and this specific page's styles are inline?

If so, when do you start thinking about extracting them? If you had more styling on this page, would you still keep it inline, since the styles apply only to this page and you can avoid an extra request, or pull it out because it's a pain having both styling and markup in the same file?

Reply:
i was trying to create three clear distinctions:
1 global site styles ( external )
2 page styles ( css in head )
3 one off overrides  ( inline )

i would extract them when it got complex 
1 a global external stylesheet
2 a external stylesheet for the page, perhaps resume.css
3 a css-in-head stylesheet for page specific overrides

Reply:
Where did this line of thinking come from? Is it something you came up with and are experimenting with, or something you have seen used elsewhere, written about, etc?

Reply:
its a bastardization of your GPS and its outlining of three categories

Reply:
So it is entirely of your own invention?

Reply:
based on GPS